tests/README.md

```diff
@@ -54,101 +55,107 @@ Local testing can be disabled via as follows:
 Test Executions
 ====
 
-All tests are located under a /tests directory in the jetty-runtime project.  Each group of tests should exist within the scope of a single maven artifact which contains a deployable docker container (local, remote, or both) and the associated test cases which follow the conventions listed further below.
+All tests are located under a `/tests` directory in the `jetty-runtime` project.  Each group of tests should exist within the scope of a single maven artifact which contains a deployable docker container (local, remote, or both) and the associated test cases which follow the conventions listed further below.
 
-> NOTE: The property ‘jetty.test.image’ being passed is analogous to the FROM Docker directive.  This serves as an effective disconnect between the jetty-runtime project and the specific docker image/tag that the respective tests will be run against.  It is absolutely possible to run tests under the /tests directory against an arbitrary image.  It is up to the user to ensure they are testing from a valid container image.
+> NOTE: The property `jetty.test.image` being passed is analogous to the FROM Docker directive.  This serves as an effective disconnect between the `jetty-runtime` project and the specific docker image/tag that the respective tests will be run against.  It is absolutely possible to run tests under the `/tests` directory against an arbitrary image.  It is up to the user to ensure they are testing from a valid container image.
 
 Local Testing
 =====
 
-From the jetty-runtime/tests directory:
+From the `jetty-runtime/tests` directory:
 
 ```
-> mvn install
+> mvn install -Djetty.test.image={repository}:{tag}
```

Comment:
`-Djetty.test.image=...` is optional. So, we should probably just keep it omitted or clarify that not specifying it is equivalent to `jetty:9.4`.